### PR TITLE
CASM-3422: Security: Container Image signature verification

### PR DESCRIPTION
**Pull request by pradeepkumargl**
# Description

This PR consists of Kyverno documentation related changes/features which are submitted to CSM 1.6 release.
Mainly Kyverno version upgrade and Container image signature verification policy features. 
This PR is for JIRA tickets
[CASM-4673](https://jira-pro.it.hpe.com:8443/browse/CASM-4673).
[CASM-4820](https://jira-pro.it.hpe.com:8443/browse/CASM-4820)

### CODE REVIEW

File: operations/kubernetes/Kyverno.md [hunk 1]
```diff
@@ -392,6 +392,413 @@ Example output:
 "cray-ceph-csi-cephfs-nodeplugin","DaemonSet","ceph-cephfs","disallow-privileged-containers","validation error: Privileged mode is disallowed. The fields spec.containers[*].securityContext.privileged and spec.initContainers[*].securityContext.privileged must be unset or set to `false`. Rule autogen-privileged-containers failed at path /spec/template/spec/containers/0/securityContext/privileged/"
 ```
 
+## What is new in the HPE CSM 1.6 release and above
+
+1. Kyverno is upgraded from 1.9.5 version to 1.10.7 version and is now available for customers as part of the HPE CSM 1.6 release.
+
+   This is a major upgrade with many new features and bug fixes. For complete list please refer to the link [CHANGELOG](https://github.com/kyverno/kyverno/blob/main/CHANGELOG.md)
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:399]:
```suggestion
   This is a major upgrade with many new features and bug fixes. For a complete list, refer to the [Kyverno CHANGELOG](https://github.com/kyverno/kyverno/blob/main/CHANGELOG.md).
```

File: operations/kubernetes/Kyverno.md [hunk 2]
```diff
@@ -392,6 +392,413 @@ Example output:
...
+
+2. Container image signing and verification using Kyverno policy.
+
+### Container image signing and verification using Kyverno policy
+
+Container images are signed and verified using a Kyverno policy for software supply chain security. For more information, refer to the link
+[Verify image signatures](https://kyverno.io/docs/writing-policies/verify-images)
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:406]:
```suggestion
[Verify image signatures](https://kyverno.io/docs/writing-policies/verify-images).
```

File: operations/kubernetes/Kyverno.md [hunk 3]
```diff
@@ -392,6 +392,413 @@ Example output:
...
+          imageReferences:
+          - '*'
+          mutateDigest: false
+          required: true
+          verifyDigest: false
+      validationFailureAction: Audit
+      webhookTimeoutSeconds: 30
+    ```
+
+    The unsigned container images added as exceptions won't be reported as policy violations in the policy report. To understand more about adding exceptions, refer to this link
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:594]:
```suggestion
    The unsigned container images added as exceptions will not be reported as policy violations in the policy report. To understand more about adding exceptions, refer to this link
```

File: operations/kubernetes/Kyverno.md [hunk 4]
```diff
@@ -392,6 +392,413 @@ Example output:
...
+    [Adding Exceptions](https://release-1-9-0.kyverno.io/docs/writing-policies/match-exclude/#match-statements)
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:595]:
```suggestion
    [Adding Exceptions](https://release-1-9-0.kyverno.io/docs/writing-policies/match-exclude/#match-statements).
```

File: operations/kubernetes/Kyverno.md [hunk 5]
```diff
@@ -392,6 +392,413 @@ Example output:
...
+          imageReferences:
+          - '*'
+          mutateDigest: false
+          required: true
+          verifyDigest: false
+      validationFailureAction: Audit
+      webhookTimeoutSeconds: 30
+    ```
+
+    The container images succesfully signed by the Customers using their own private key, won't be reported as policy violations in the policy report.
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:721]:
```suggestion
    The container images successfully signed by the Customers using their own private key, will not be reported as policy violations in the policy report.
```

File: operations/kubernetes/Kyverno.md (hunk 1 above)
Comment (rustydb) [reply operations/kubernetes/Kyverno.md:399]: *Kyverno

**Pull request closed without merging**